### ISSUE TITLE
Home/Search: English text on the French page

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#51]
URL: https://hcdev.openplus.ca/fr; https://hcdev.openplus.ca/fr/user/203/search
User Agent: Chrome, Firefox
Language: French

Issue Description: There is some English on the French page. Page title of Home is in English. The page listing at the bottom has English tooltip and the "Last" is in English. "First" is also in English. 

Expected results: Text should be in French as these are French pages. These are from Home and from Search page. Please make sure if there are other pages with similar pagination, they are in correct languages. 

![image](https://user-images.githubusercontent.com/14280062/128758893-9b7ac092-aa01-4a94-8b97-7d36b71e6b69.png)

![image](https://user-images.githubusercontent.com/14280062/128757931-b03eddaa-b5a1-428b-b1ea-d55d92953a07.png)

![image](https://user-images.githubusercontent.com/14280062/128758241-8bd19274-82e7-490f-85bd-8c3f31218005.png)

**Comment by davidgotlieb** [comment]
﻿This is fixed in d9 update.

**Comment by ndolter** [comment]
The French page now has the translations, however, a couple of issues still remain:

1. The button labels for 'Premiere' and 'Dernier' are spelled incorrectly. Please note that these button labels in both languages do not match to those in WET.  
WET = 'Précédent' and 'Suivant' / 'Previous' and 'Next'
RSAMS = 'Premiere' and 'Dernier' / 'First' and 'Last'

2. WET does not include a tooltip (e.g. Go to page X') as the screen reader already reads 'Page X' link. See:  https://wet-boew.github.io/v4.0-ci/demos/tables/tables-fr.html

![image](https://user-images.githubusercontent.com/68117281/131528385-0433590f-c7dd-4137-99f9-438dbceca377.png)

3. The Page title of Home | RSAMS has not been translated and should match to the H1: 'Trouvez des rappels, des avis et des avis de sécurité'. The English page would also need to be fixed to change from 'Home' to 'Find recalls, advisories and safety alerts.' in order to properly describe the page.

**Comment by echouin** [comment]
OP confirmed: fixed.

**Comment by qanitah** [comment]
Fixed
